### PR TITLE
Update README.rst (#75)

### PR DESCRIPTION
**Pull request by niccokunzmann**
Docs update

### CODE REVIEW

**Comment by niccokunzmann** [comment]
@MarkoMackic If you like, you can merge this and it should go live within minutes.

**Comment by artemrizhov** [comment]
I was commenting on this pull request early. Has anyone seen my comments?

File: README.rst [hunk 1]
```diff
@@ -42,12 +42,13 @@ You may also pass additional parameters.
 
 .. code:: python
 
-    monitoring_thread = start_monitoring(seconds_frozen=10, tests_per_second=10)
+    monitoring_thread = start_monitoring(seconds_frozen=10, test_interval=100)
 
 The values in the example are defaults. This mean the check will happen 10
 times per second. If a thread is frozen for at least 10 seconds then the stack
```
Comment (artemrizhov) [inline README.rst:48]: > This mean the check will happen 10 times per second.

This text is about `tests_per_second` arg. What if we change this with something like "check will happen every 100 milliseconds"?

File: README.rst [hunk 2]
```diff
@@ -42,12 +42,13 @@ You may also pass additional parameters.
...
 is dumped into standard error stream. This happens again every 10 seconds
-while there is no changes in the stack registered during checks.
+while there is no changes in the stack registered during checks. Checks are done
+every `test_interval` milliseconds, default is 100.
```
Comment (artemrizhov) [inline README.rst]: The text above is already saying "The values in the example are defaults". Also if we replace "check will happen 10 times per second" with more appropriate text then this change is probably will not be needed. What do you think about this?

**Review by artemrizhov — DISMISSED** [review]

File: README.rst (hunk 2 above)
Comment (MarkoMackic) [inline README.rst]: Nice I'll change it

Comment (MarkoMackic) [reply README.rst]: How should we form this, just update `check will be done 10 times per second` and not repeat that it's default. but then they won't know what exactly parameter defines.

**Comment by niccokunzmann** [comment]
I made it an issue #80.

File: README.rst (hunk 2 above)
Comment (artemrizhov) [reply README.rst]: @MarkoMackic This is the form that has been used already, so this kind of fix seems correct. But maybe thext text is not really clear. If you think so then this text probably should be completely rewritten.

**Comment by niccokunzmann** [comment]
@MarkoMackic If you like, you can create a new pull-request into the v2.0 branch to create a new version on Pypi.

**Comment by MarkoMackic** [comment]
Sure

**Pull request closed without merging**